### ISSUE TITLE
Clipboard is not working on Ubuntu 18.04 LTS on server Version 0.4.3-x

### ISSUE THREAD

**Issue by southeasterntech** [Ylianst/MeshCentral#646]
Multiple attempts , reinstalling 3 separate times, the clipboard functionality does not seem to work.  please confirm.  It says successful when copying content over, but Paste is not available in terminal.

**Comment by krayon007** [comment]
Clipboard on Linux is finicky, because it's using the X11 clipboard. If the app you are pasting into doesn't use the X clipboard, paste will not work.

**Comment by southeasterntech** [comment]
it's just regular terminal on Linux? (In this case)

**Comment by krayon007** [comment]
I'll take a look at it when I get back from holiday.

**Comment by southeasterntech** [comment]
thanks!  Just trying to do a install of some software with tons of commands and I have to do it from the mesh desktop instead of from my workstation (copy\paste)

**Comment by krayon007** [comment]
Just to clarify, are you talking about terminal inside remote desktop, or are you talking about terminal, as in the terminal tab on the web portal?

**Comment by southeasterntech** [comment]
Ah... terminal in remote desktop.... say when I am on the Ubuntu remote desktop, I launch terminal, and then in the Ubuntu Desktop, I use the Clipboard button on my mesh Windows workstation, paste content, choose "set clipboard" , it says "Success" in green, then I right click on the terminal window and the Paste option is grayed out.  Ctrl-V also doesn't work.  I've scratch reinstalled the Ubuntu VM and reinstalled mesh, 3 times with the same result, so I suspect it's not limited to one case.
Thanks.

**Comment by krayon007** [comment]
Ok, I did some testing and googling, and found that this is a known issue with the terminal emulator on linux, as the terminal uses its own cutbuffer, not the X11 selection buffer... However, the solution is easy...

When pasting into the terminal, use shift-middle-click. I tested it on my Ubuntu 18.04, and it worked like a champ. I set the clipboard from the webpage, and held the shift key while middle clicking my mouse in the terminal, and it pasted from the X11 selection buffer.
